eea/dexterity/indicators/behaviors/indicator.py

```diff
@@ -19,6 +19,12 @@ def getAllBlocks(blocks, flat_blocks):
     return flat_blocks
 
 
+def dedupe_data(data):
+    """Remove duplication from metadata fields"""
+    res_list = [i for n, i in enumerate(data) if i not in data[n + 1:]]
```

Comment:
@nileshgulia1 I like this ingenious way of finding duplicates. 

Unfortunately, as the list elements are dictionaries it doesn't seem to work in all cases. 

Thus, I suggest you dive into these dictionaries, extract the url and compare them. instead of the entire dict element.

And don't forget `The Zen of Python`

```
>>> import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
...

```

Reply:
@nileshgulia1 True but in our case we're comparing organizations which should have the same `Title -> URL` pair, thus it doesn't make sense to have 2 texts pointing to the same URL.

On the other hand, the above code will fail if you have the same text and url, but one bold, one italic and one normal.

For this, I think it's better to consider URL as unique id and compare based on them.